translations/SECURITY.portuguese_br.md

```diff
@@ -0,0 +1,39 @@
+## Reportando Vulnerabilidades
+
+Nós somos extremamente gratos a pesquisadores de segurança e usuários que reportam vulnerabilidades para a Comunidade Hasura. Todos os laudos são minuciosamente investigando por um conjunto de membros voluntários da comunidade além do time da Hasura.
+
+Para reportar um problema de segurança, por favor mande um e-mail para [build@hasura.io](mailto:build@hasura.io) (em inglês) com todos os detalhes, anexando todas informações necessárias.
+
+### Quando eu devo reportar uma vulnerabilidade?
+
+- Você acha que encontrou uma potencial vulnerabilidade de segurança na Engine GraphQL Hasura ou algum componente relaiconado.
+- Você não tem certeza de como uma vulnerabilidade afeta a Engine GraphQL Hasura.
+- Você acha que descobriu uma vulnerabilidade em outro projeto que a Engine GraphQL Hasura depende (e.x. Heroku, Docker, etc).
+- Você quer reportar qualquer outro risco de segurança que pode, potencialmente, ser malicioso para os usuários da Engine GraphQL Hasura.
+
+### Quando eu NÃO devo reportar uma vulnerabilidade?
+
+- Você precisa de ajuda para configurar os componentes da Engine GraphQL Hasura para segurança.
+- Você precisa de ajuda para aplicar atualizações relacionadas à segurança.
+- Você tem um problema não relacionado a segurança.
+
+## Resposta às vulnerabilidades de segurança
+
+Cada relatório é reconhecido e analizado pelos mantenedores do projeto e o time de segurança dentro de 3 dias de trabalho.
+
+O relator será mantido atualizado em cada estágio do processo de análise do problema e de sua resolução (triagem -> correção -> release/lançamento).
+
+## Atraso para Divulgação Pública
+
+Uma data de divulgação pública é negociada entre o time de segurança do produto Hasura e o relator. Nós preferimos divulgar totalmente o bug o mais cedo possível assim que tivermos uma mitigação para o problema. É razoável atrasar a divulgação quando o bug ou a correção ainda não são totalmente compreendidos, a solução ainda não foi bem testada, ou para coordenação de terceiros. O prazo para divulgação pode variar entre imediato (especialmente quando for um problema já publicamente conhecido) até algumas semanas. Nós esperamos que o time entre o relatório e a divulgação pública fique tipicamente em torno de 7 dias. Os mantenedores da Engine GraphQL Hasura e o time de segurança tomam a decisão final em escolher uma data de divulgação.
```

Comment:
Uma data de divulgação pública é negociada entre o time de segurança do produto Hasura e o relator. Preferimos divulgar o bug por completo o mais cedo possível assim que tivermos uma solução para o problema. É compreensível atraso na divulgação quando o bug ou a correção ainda não são totalmente compreendidos, a solução ainda não foi bem testada, ou é necessária a coordenação de terceiros. O prazo para divulgação pode variar entre imediato (especialmente quando for um problema já publicamente conhecido) a até algumas semanas. Esperamos que o tempo entre o relatório e a divulgação pública fique em torno de 7 dias, preferencialmente. Os mantenedores da Engine GraphQL Hasura e o time de segurança são quem decide sobre a data final de divulgação.